tox.ini

```diff
@@ -57,7 +60,7 @@ commands =
     touch tests/__init__.py
     pylint enterprise
     pylint integrated_channels
-    pylint tests
+    pylint tests --load-plugins=pylint.extensions.redefined_variable_type
```

Comment:
@snowcrshd this could go into tests/pylintrc (if I recall right that you can specify plugins via pylintrc)